magefiles/lint.go

```diff
@@ -0,0 +1,35 @@
+//go:build mage
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/magefile/mage/mg"
+)
+
+type Lint mg.Namespace
+
+// All Run all linters
+func (l Lint) All() error {
+	mg.Deps(l.Go)
+	return nil
+}
+
+// Go Run all go linters
+func (l Lint) Go() error {
+	mg.Deps(l.Gofumpt, l.Golangcilint)
+	return nil
+}
+
+// Gofumpt Run gofumpt
+func (Lint) Gofumpt() error {
+	fmt.Println("formatting go")
+	return RunSh("go", Tool())("run", "mvdan.cc/gofumpt", "-l", "-w", "..")
+}
```

Comment:
I know this is how we're doing it in other repos, but can we make this the first place where we use `golangci-lint fmt` instead?

Reply:
Easy enough to just do now